api/models/cronjob.go

```diff
@@ -0,0 +1,107 @@
+package models
+
+import (
+	"archive/zip"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type CronJob struct {
+	Name          string `yaml:"name"`
+	Schedule      string `yaml:"schedule"`
+	Command       string `yaml:"command"`
+	ManifestEntry *ManifestEntry
+}
+
+func NewCronJobFromLabel(key, value string) CronJob {
+	keySlice := strings.Split(key, ".")
+	name := keySlice[len(keySlice)-1]
+	tokens := strings.Fields(value)
+	cronjob := CronJob{
+		Name:     name,
+		Schedule: fmt.Sprintf("cron(%s *)", strings.Join(tokens[0:5], " ")),
+		Command:  strings.Join(tokens[5:], " "),
+	}
+	return cronjob
+}
+
+func (cr *CronJob) AppName() string {
+	return cr.ManifestEntry.app.Name
+}
+
+func (cr *CronJob) Process() string {
+	return cr.ManifestEntry.Name
+}
+
+func (cr *CronJob) ShortName() string {
+	return fmt.Sprintf("%s%s", strings.Title(cr.ManifestEntry.Name), strings.Title(cr.Name))
+}
+
+func (cr *CronJob) LongName() string {
+	app := cr.ManifestEntry.app
+	return fmt.Sprintf("%s-%s-%s", app.StackName(), cr.Process(), cr.Name)
+}
+
+func (cr *CronJob) UploadLambdaFunction() error {
```

Comment:
@nzoschke Thanks for the tip. I was assuming the S3 bucket referred to [here](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-code.html) had to be in the same account, but maybe not. If I could just point to a zip file in a public bucket that would be fantastic.

The only unique thing in my current lambda function is the rack stack name, but I can probably configure the CloudWatch Event to send that, too.

See my comment above about inlining. As I read the docs, that's not going to be possible after October 2016.